Woof/Woof/Model/Sitter/Sitter.swift

```diff
@@ -29,4 +29,62 @@ struct Sitter: Codable, Identifiable {
 
     /// The price per hour charged by the sitter.
     var pricePerHour: Double
+
+    /**
+     Creates a pet sitter instance with the specified parameters with the unique identifier.
+
+     - Parameters:
+     - name: The first name of the pet sitter.
+     - surname: The last name of the pet sitter.
+     - phone: The phone number of the pet sitter.
+     - avatarURL: The URL that provides an avatar image of the pet sitter.
+     - bio: The additional information about the pet sitter.
+     - rating: The rating of the pet sitter.
+     - pricePerHour: The price per hour charged by the pet sitter.
```

Comment:
```suggestion
       - name: The first name of the pet sitter.
       - surname: The last name of the pet sitter.
       - phone: The phone number of the pet sitter.
       - avatarURL: The URL that provides an avatar image of the pet sitter.
       - bio: The additional information about the pet sitter.
       - rating: The rating of the pet sitter.
       - pricePerHour: The price per hour charged by the pet sitter.
```
The parameter names don't look properly in preview. 
<img width="486" alt="Screenshot 2023-06-19 at 12 31 46" src="https://github.com/ios-course/ironfoudation-team-project/assets/110284165/fc8801be-37d7-49d8-ac71-3907c817db01">

Reply:
This file isn't a part of this PR. It was created before.